examples/kotlin-diktat/warn-dir/chapter2/save.toml

```
@@ -2,8 +2,9 @@
 tags = "tag1"
 description = "chapter 2 tests: with default lines (no line numbers)"
 suiteName = "Chapter2"
-expectedWarningsPattern = "// ;warn:(\\d+) (.+)"
+expectedWarningsPattern = "// ;warn:(\\d*):(\\d*): (.+)"
```

```suggestion
expectedWarningsPattern = "// ;warn:(\\d*):?(\\d*): (.+)"
```
I believe we can remove duplication of colons with something like this

---

Thanks! Fixed